frontend/src/components/Footer.css

```diff
@@ -0,0 +1,58 @@
+.footer {
+	margin: 0 48px 24px 48px;
+}
+
+.footer-1, .footer-2, .footer-3 {
```

Comment:
we should always aim to write Self Documenting Code, and one tenet of that is good naming (though this is one of the hardeset parts of coding). footer-1 gives me a clue but doesn't really tell me all that much otherwise about its intended usage. can we use better names here?


related reading: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/